src/services/loading-files.js

```diff
@@ -41,29 +41,43 @@ export function createExecutableFromProject(project) {
  */
 export function createProjectStructure(mainFile) {
   let modules = findRequires(mainFile)
+  let orderedModNames = modules.map((m) => m.name)
 
   for (let i = 0; i < modules.length; i++) {
-    if (modules[i].content || !fs.existsSync(modules[i].path)) continue
```

Comment:
Checking if the module exists needs to stay. For e.g. if the user does `require("json")`, this will throw an error, because the json module is not present, even tho it is part of the ao environment.

Reply:
Also I believe often one module can be required by multiple other modules, so we need to check if `modules[i].content` is not defined, because if it is, we don't want to it twice.

Reply:
So we'll need this entire line back
```suggestion
    if (modules[i].content || !fs.existsSync(modules[i].path)) continue
```